packages/webview_flutter/example/lib/main.dart

```diff
@@ -111,6 +120,17 @@ class SampleMenu extends StatelessWidget {
       ),
     );
   }
+
+  void _onClearCookies(BuildContext context) async {
+    final bool hadCookies = await cookieManager.clearCookies();
+    String message = "There were cookies. Now, they are gone!";
```

Comment:
How do we know that it worked?
Maybe we could make a JS call to get the list of cookies present it?

Reply:
Good point, for now i verified that flutter.io was setting and deleting cookies. Will add a method to list all cookies.